### PR TITLE
salesgpt incompleto

### PR DESCRIPTION
**Pull request by thalex**

### CODE REVIEW

File: make_call.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Download the helper library from https://www.twilio.com/docs/python/install
+import os
+from twilio.rest import Client
+
+
+
+# Find your Account SID and Auth Token at twilio.com/console
+# and set the environment variables. See http://twil.io/secure
+account_sid = 'AC4ccd7e15f30bc287ef5bb019cedd0c0a'
+auth_token = '91b22def36f592eb2bef7c56eb62b237'
```
Comment (filip-michalsky) [inline make_call.py:10]: Please remove your personal token and sid otherwise it's going to run up your bill substantially :)

**Review by filip-michalsky — COMMENTED** [review]
This is definitely WIP - but a good start, can you please let me know the flow you are trying to build here? Thank you!

File: make_call.py (hunk 1 above)
Comment (ranjandsingh) [reply make_call.py:10]: @thalex  Dude remove your keys or change the token man

**Comment by filip-michalsky** [comment]
@thalex its been a while, I am going to close this and reopen if/when you get to it

**Pull request closed without merging**